exercises/pascals-triangle/pascals_triangle_test.exs

```diff
@@ -0,0 +1,44 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("pascals_triangle.exs")
+end
+
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule PascalsTriangleTest do
+  use ExUnit.Case
+
+  test "test one row" do
```

Comment:
@dalexj we've been putting a commented out `@tag pending` on the first spec as an example for submitters. I think we should be consistent.